content/doc/installation/with-docker.en-us.md

```diff
@@ -17,111 +17,179 @@ menu:
+
+## Basics
+
+The most simple setup just creates a volume and a network and starts the `gitea/gitea:latest` image as a service. Since there is no database available you can start it only with SQLite3. Create a directory like `gitea` and paste the following content into a file named `docker-compose.yml`.
+
+```yaml
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
+    restart: always
+    networks:
+      - gitea
+    volumes:
+      - ./gitea:/data
+    ports:
+      - 3000:3000
+      - 222:22
+```
+
+## Custom port
+
+To bind the integrated openSSH daemon and the webserver on a different port, you just need to adjust the port section. It's common to just change the host port and keep the ports within the container like they are.
+
+```diff
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
+    restart: always
+    networks:
+      - gitea
+    volumes:
+      - ./gitea:/data
+    ports:
+-      - 3000:3000
+-      - 222:22
++      - 8080:3000
++      - 2221:22
+```
+
+## MySQL database
+
+To start Gitea in combination with a MySQL database you should apply these changes to the `docker-compose.yml` file created above.
+
+```diff
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
+    restart: always
+    networks:
+      - gitea
+    volumes:
+      - ./gitea:/data
+     ports:
+       - 3000:3000
+       - 222:22
++    depends_on:
++      - db
++
++  db:
++    image: mysql:5.7
++    restart: always
++    environment:
++      - MYSQL_ROOT_PASSWORD=gitea
++      - MYSQL_USER=gitea
++      - MYSQL_PASSWORD=gitea
++      - MYSQL_DATABASE=gitea
++    networks:
++      - gitea
++    volumes:
++      - ./mysql:/var/lib/mysql
+```
+
+## PostgreSQL database
+
+To start Gitea in combination with a PostgreSQL database you should apply these changes to the `docker-compose.yml` file created above.
+
+```diff
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
+    restart: always
+    networks:
+      - gitea
+    volumes:
+      - ./gitea:/data
+     ports:
+       - 3000:3000
+       - 222:22
++    depends_on:
++      - db
++
++  db:
++    image: postgres:9.6
++    restart: always
++    environment:
++      - POSTGRES_USER=gitea
++      - POSTGRES_PASSWORD=gitea
++      - POSTGRES_DB=gitea
++    networks:
++      - gitea
++    volumes:
++      - ./postgres:/var/lib/postgresql/data
+```
+
+## Named volumes
+
+To use named volumes instead of host volumes you just have to define and use the named volume within the `docker-compose.yml` configuration. This change will automatically create the required volume.
+
+```diff
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
++volumes:
++  gitea:
++    driver: local
```

Comment:
Is not this default?

Reply:
Same as above, something like just `gitea:` looks for beginners incomplete, that's why I add a simple option that doesn't hurt.